android/daemon/build.gradle

```diff
@@ -52,6 +41,8 @@ android {
         minSdkVersion Config.minSdkVersion
         targetSdkVersion Config.targetSdkVersion
 
+        buildConfigField "int", "VERSION_CODE", System.getenv("VERSIONCODE") ?: "99999"
+        buildConfigField "String", "VERSION_NAME", '"' + (System.getenv("SHORTVERSION") ?: "") + '"'
         buildConfigField  "String", "VERSIONCODE" ,  '"' +System.getenv("VERSIONCODE") + '"'
```

Comment:
Why twice?

Reply:
Oh, one is "VERSIONCODE" and the other is "VERSION_CODE" 👀 

It's good you commented here. It reminds me I had a little issue with these variables. 

You and the internet had told me I should use `versionName` and `versionCode` here and they would set these variables. That didn't work though, that is why I used `buildConfigField` here instead.

Also, I set these in a bunch of places, but I could not figure out how to just set it once. I don't like the way it is right now :(